lib/config/configuration.js

```diff
@@ -949,6 +949,8 @@ Configuration.prototype.registerDefaultRules = function() {
     this.registerRule(require('../rules/require-matching-function-name'));
 
     this.registerRule(require('../rules/disallow-space-before-semicolon'));
+
+    this.registerRule(require('../rules/validate-order-in-object-keys'));
```

Comment:
Yeah right now adding it to the end is fine - maybe someone can do a separate PR to sort it at some point.